CMakeLists.txt

```diff
@@ -212,29 +140,26 @@ else()
   message(FATAL_ERROR "Required target protobuf::libprotobuf-lite or protobuf::libprotobuf is not defined")
 endif()
 
-add_library(libprime_server INTERFACE IMPORTED)
+# libprime_server
+# see https://gitlab.kitware.com/cmake/cmake/-/issues/19467
+set(libprime_server_targets "")
```

Comment:
I think it's better to do this `set(... )` here than `add_library()`

Reply:
is that because you can see the thing that is set versus that being implicit when calling `add_library`?